src/app/components/mnoe-users-local-list/mnoe-users-local-list.coffee

```diff
@@ -78,25 +79,17 @@
           MnoErrorsHandler.processServerError(error)
       ).finally(-> user.isSendingInvite = false)
 
-    scope.availableRoles = () ->
-      [
-        "mnoe_admin_panel.constants.user_roles.member",
-        "mnoe_admin_panel.constants.user_roles.admin",
-        "mnoe_admin_panel.constants.user_roles.super_admin"
-      ]
-
-    scope.indexOfRoleInTable = (role) ->
-      scope.availableRoles().indexOf(scope.tlKeyFromUserRole(role))
-
     # role here should always be in English (as returned by MnoHub)
+    # This method returns the translation key associated to this role,
+    # for internationalization
```

Comment:
just a side note for the future, i18n instead of internationalization works too :)